test/type-traits.cc

```diff
@@ -108,3 +109,41 @@ static_assert(
         std::tuple<int>>);
 
 ////////////////////////////////////////////////////////////////////////
+
+static_assert(std::is_same_v<
```

Comment:
```suggestion
static_assert(
    std::is_same_v<
```
Let's use the same formatting as above to be consistent in the file. Here and the rest of what you added please.